### PR TITLE
add :  "Project" add Japanese translation to i18n/ja

### PR DESCRIPTION
**Pull request by arisamikumo**
## description
I translated Project page to Japanese freely.

## Concern

1. **I added "ja" at the beginning of  all links in this page.→it might not exist yet.**  

2. **I translated words like below:**
  - Archive：非編集化
  - Editing page：編集画面
  - Public status：公開状況
  - Cooperate：共同編集
  - Current Project：現在のプロジェクト
  →need Michi-san to check those translation. Cuz she knows the words using in Re:Earth well.

### CODE REVIEW

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 1]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
 
-The following operations are available on a project:
+プロジェクトでは以下の操作が可能です：
 
-- [Create](/user-manual/project-and-workspace/project/create-a-new-project)
-- [Preview]
-- [Publish]
-- [Archive](/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
-- [Delete](/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
+- [作成](ja/user-manual/project-and-workspace/project/create-a-new-project)
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:21]: please don't add "ja" in front of the link, the system know it is on "/js", which is automatically。

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 2]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
+- [プレビュー]
+- [公開]
+- [非編集化](ja/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:24]: here also

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 3]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
+- [削除](ja/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:25]: 🙏

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 4]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
+    <strong>現在のプロジェクト</strong> プロジェクトの基本情報の変更や、プロジェクトの非編集化や削除を行うことができます。詳しくは、<Link to="ja/user-manual/project-and-workspace/project/settings-archiving-and-deleting">設定・非編集化・削除</Link>をご覧ください。
+  </List>
   <List>
-    <strong>Public</strong> settings page provides some settings about publishing your project. For more details, please check <Link to="/user-manual/project-and-workspace/project/public-settings">Public Setting</Link>.
+    <strong>公開設定</strong> プロジェクトの公開に関する設定を行うことができます。詳しくは、<Link to="ja/user-manual/project-and-workspace/project/public-settings">公開設定</Link>をご覧ください。
   </List>
   <List>
-    <strong>Dataset</strong> library page is used to manage the datasets that have been imported into the current project. For more details, please check <Link to="/user-manual/project-and-workspace/project/dataset-library">Dataset library</Link> and [the Dataset section].
+    <strong>データセット</strong> 現在のプロジェクトにインポートされているデータセットを管理することが出来ます。詳しくは、<Link to="ja/user-manual/project-and-workspace/project/dataset-library">データセットライブラリ</Link> および [データセットセクション]をご覧ください。
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:64]: 🙏

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 5]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
   </List>
   <List>
-    <strong>Plugin</strong> library page is used to manage the installed plugins of the current project. For more details, please check <Link to="/user-manual/project-and-workspace/project/plugin-library">Plugin Library</Link> and the entire [Plugin section].
+    <strong>プラグイン</strong> 現在のプロジェクトにインポートされているプラグインを、管理することが出来ます。詳しくは、<Link to="ja/user-manual/project-and-workspace/project/plugin-library">プラグインライブラリ</Link> および [プラグインセクション]をご覧ください。
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:67]: 🙏

File: i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx [hunk 6]
```diff
@@ -3,88 +3,90 @@ title: 概要
...
-In the upper right corner of the Editor page you can also see and change the status at any time while editing.
+編集画面の右上に、公開状況が表示されおり、いつでも確認・変更することが可能です。
 
 <Image path={require('./img/1-7.png').default} />
 
-## Cooperate
+## 共同編集
 
-In order to support collaborative editing between team members, we have developed a workspace system. Members in the same workspace can participate in editing the same project. For specific operations, please refer to the [workspace](/user-manual/project-and-workspace/workspace/overview) section.
+ワークスペースシステムにより、チームメンバー間の共同編集が可能になります。同じワークスペースにいるメンバーは同じプロジェクトを編集することが出来ます。具体的な操作方法については、[ワークスペース](ja/user-manual/project-and-workspace/workspace/overview) をご覧ください。
```
Comment (lavalse) [inline i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx:89]: 🙏

**Comment by arisamikumo** [comment]
## Creating a new project
- I think the current order can be improve. (Hard to write in words, so wanna discuss💬)
- project description explained it is helpful to discern similar projects when the number of projects is large.  But I'm not sure about this. so I didn't translate that part for now. Wanna discuss about this.💬
- "Create button" wrote in "D" of Project creation modal is not "`Create` button"→This is because it's in <order>or<list>tag?
- concern：Images are English

**Comment by arisamikumo** [comment]
## Setting, archiving and deleting 
- is archiving correct translation  非編集化 or アーカイブ??(Dangerous zone the bottom of setting and `unarchive` button said アーカイブ, but "archive" button said 非編集化 )
- Politeness regarding operating instructions is different between "create new project"and "archived project"→There is no explanation how to access the setting page to archive or unarchive the project.
- should add “unarchive project”???💬
- caution should be 注意 if it can be

**Comment by arisamikumo** [comment]
## Public setting
- Just so you know, link has not set yet.
- wanna make sure if my understanding about each setting options is correct or not

**Comment by arisamikumo** [comment]
## Dataset Library
- I think it's okay

**Review by lavalse — APPROVED** [review]

**Pull request merged**